php/api.php

```diff
@@ -0,0 +1,63 @@
+<?php
+header('Content-type:application/json');
+
+$signingSecret = '<SIGNING_SECRET>';
+$requestPath = getRequestPath();
+
+if ($requestPath[0] !== 'webhook') {
+  response(404, 'Route not found', null);
+  exit();
+}
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+  $body = file_get_contents('php://input');
```

Comment:
Let's leave a comment here about this line:
```
// Use the request body string exactly as provided without and transformations for the signature check.
// If this data is transformed, decoded, encode in anyway it will fail the signature check. 
$body = file_get_contents('php://input');
```